.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5503
```

Comment:
- [ ] Kindly remove `.vscode` from your code directory to make your project clean.